DataFormats/L1TCorrelator/interface/TkHTMiss.h

```diff
@@ -7,7 +7,7 @@
 #include "DataFormats/Common/interface/Ref.h"
 #include "DataFormats/Common/interface/RefProd.h"
 #include "DataFormats/Common/interface/Ref.h"
-#include "DataFormats/L1TCorrelator/interface/TkPrimaryVertex.h"
+#include "DataFormats/L1Trigger/interface/Vertex.h"
```

Comment:
Just curiosity, what is the difference between old one and new one? Later of this PR (L1Trigger/L1TTrackMatch/plugins/L1TkHTMissEmulatorProducer.cc), you still use old one (TkPrimaryVertex.h).

Reply:
@BenjaminRS Can you please comment?

Reply:
Hi @srimanob and @cecilecaillol. The TkPrimaryVertex is an old version of the vertexing. I believe it should be removed but I need to check further and confer with colleagues to make sure it is safe to do so.

Reply:
Hi @BenjaminRS 
Thanks for clarification. I'm fine with the PR if you would like to clean it up later. Please make a github issue, so that we can follow.
By the way, `L1Trigger/L1TTrackMatch/plugins/L1TkHTMissEmulatorProducer.cc` should be fixed?

Reply:
Hi @cecilecaillol and @srimanob - I have investigated a bit why the TkPrimaryVertex had been kept around; it was due to the HLT P2 TDR. I have contacted the HLT P2 coordinators to see if we can go ahead and drop it.